app/views/start_page.scala.html

```diff
@@ -14,14 +14,12 @@
  * limitations under the License.
  *@
 
+@import config.AppConfig
 @import controllers.routes
 
 @this(main_template: views.html.main_template)
 
-@()(implicit request: Request[_], messages: Messages)
-
-@customsDeclarationsOutOfEuUrl = {"https://www.gov.uk/guidance/customs-declarations-for-goods-taken-out-of-the-eu"}
-@serviceAvailabilityUrl = {"https://www.gov.uk/guidance/customs-declaration-service-service-availability-and-issues"}
+@()(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
```

Comment:
why not inject app config directly to view ?

Reply:
Did not think about it. It is injected now.